service/storage_service.cc

```diff
@@ -654,6 +626,19 @@ void storage_service::join_token_ring(int delay) {
         }
     }
 
+
+    if (!is_auto_bootstrap()) {
+        slogger.warn("auto_bootstrap set to \"off\". This causes UNDEFINED BEHAVIOR. YOU WILL LOSE DATA.");
```

Comment:
Why this?

Reply:
For the same reason. auto_bootstrap=off does the same thing as marking the node as seed.

Reply:
s/YOU WILL LOSE DATA/YOU MAY LOOSE DATA/

Reply:
I prefer "YOU WILL LOSE DATA", it's more dramatic. Scare them, because they should be scared.
But of course it's not completely true :D so I'll change it.